src/main/java/com/sprint_manager/repos/SprintRepo.java

```diff
@@ -0,0 +1,106 @@
+        return jdbcTemplate.query(
+                "select sprints.id, users.full_name, sprints.title, sprints.state, " +
+                        "sprints.date_start, sprints.date_end_expect, sprints.date_end_fact" +
+                        "  from sprints inner join users on sprints.author_id = users.id",
+                new SprintMapper()
+        );
+    }
+
+    public Integer getCountOfTasks(Integer id) {
+        return jdbcTemplate.queryForObject(
+                "select count(*) from sprint_backlog where sprint_id = ?",
+                new Object[] { id },
+                Integer.class
+        );
+    }
+
+    public Integer insertNewTask(Sprint sprint) {
+        return jdbcTemplate.queryForObject(
+                "insert into sprints (title, author_id, state, date_start, date_end_expect) " +
+                        "values(?, ?, ?, ?, ?) returning id",
+                new Object[] {
+                        sprint.getTitle(),
+                        sprint.getAuthorId(),
+                        sprint.getSprintState(),
+                        sprint.getStartDate(),
+                        sprint.getEndDateExpect()
+                },
+                Integer.class
+        );
+    }
+
+    public Sprint getSprintById(Integer id) {
+        return jdbcTemplate.queryForObject(
+                "select s.id, u.full_name, s.title, s.state, " +
+                        "s.date_start, s.date_end_expect, s.date_end_fact " +
+                        "from sprints as s inner join users as u on s.author_id = u.id " +
+                        "where s.id = ?",
+                new Object[] {id},
+                new SprintMapper()
+        );
+    }
+
+    public void updateSprintStateAndActEndDate(Integer id, String state, Date dateEndFact) {
+        jdbcTemplate.update(
+                "update sprints set state = ?, date_end_fact = ? where id = ?",
+                state,
+                dateEndFact,
+                id
+        );
+    }
+
+    public Sprint getActiveSprint() {
+        try {
+            return jdbcTemplate.queryForObject(
+                    "select s.id, u.full_name, s.title, s.state, " +
+                            "s.date_start, s.date_end_expect, s.date_end_fact " +
+                            "from sprints as s inner join users as u on s.author_id = u.id " +
+                            "where s.state = ?",
+                    new Object[] { SprintState.STARTED.toString() },
+                    new SprintMapper()
+            );
+        } catch (IncorrectResultSizeDataAccessException e) {
+            return null;
+        }
+    }
+
+    public boolean updateSprint(Sprint sprint) {
+        return jdbcTemplate.update(
+                "update sprints set title = ?, date_start = ?, date_end_expect = ? where sprint.id = ?",
+                sprint.getTitle(),
+                sprint.getStartDate(),
+                sprint.getEndDateExpect(),
+                sprint.getId()
+        ) > 0;
```

Comment:
isn't it better to check if only 1 row was updated? or can it be that more then one row was updated by this query?